lib/counter_culture/counter.rb

```diff
@@ -68,18 +68,19 @@ def change_counter_cache(obj, options)
           klass = relation_klass(relation, source: obj, was: options[:was])
           primary_key = relation_primary_key(relation, source: obj, was: options[:was])
 
+          klass.where(primary_key => id_to_change).update_all updates.join(', ')
```

Comment:
@magnusvk I don't think we want to move this before the papertrail call, because that defeats the purpose of creating the version row. It's precisely because the `update_all` does not trigger any version rows to be created which is why we're forcing one before the update happens (in order to record the state of the object before it gets updated).

In our local specs that test this behavior, we compare counted_class.versions.last.object["counted_column_name"] to what it was *before* the update_all.